community/io/src/main/java/org/neo4j/io/fs/StoreFileChannel.java

```diff
@@ -170,4 +170,10 @@ public void flush() throws IOException
     {
         force( false );
     }
+
+    public static FileChannel unwrap( StoreChannel channel )
```

Comment:
Because it's not part of the "normal" API. It is a method that circumvents the encapsulation of the StoreFileChannel.